modules/snooker/src/test/ts/browser/model/TransitionsTest.ts

```diff
@@ -0,0 +1,316 @@
+import { context, describe, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { Attribute, SugarElement, TextContent } from '@ephox/sugar';
+import { assert } from 'chai';
+
+import * as Structs from 'ephox/snooker/api/Structs';
+import { Warehouse } from 'ephox/snooker/api/Warehouse';
+import * as Transitions from 'ephox/snooker/model/Transitions';
+import * as Bridge from 'ephox/snooker/test/Bridge';
+
+describe('TransitionsTest', () => {
+  const createCell = (text: string): SugarElement<HTMLTableCellElement> => {
+    const elem = SugarElement.fromTag('td');
+    TextContent.set(elem, text);
+    return elem;
+  };
+
+  const createCol = (id: string): SugarElement<HTMLTableColElement> => {
+    const elem = SugarElement.fromTag('col');
+    Attribute.set(elem, 'data-id', id);
+    return elem;
+  };
+
+  const re = () => SugarElement.fromTag('tr');
```

Comment:
I don't know how much of this is copied from the old test (it won't show me the diff because the change is too big) but if this is new code are we able to use better variable names?

Reply:
These variable names are existing and are common in the snooker tests, though I wouldn't say they are great.